x/globalfee/migrations/v2/migrate.go

```diff
@@ -0,0 +1,52 @@
+package v2
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/CosmosContracts/juno/v16/x/globalfee/keeper/exported"
+	"github.com/CosmosContracts/juno/v16/x/globalfee/types"
+)
+
+const (
+	ModuleName = "globalfee"
+)
+
+var ParamsKey = []byte{0x00}
+
+// Migrate migrates the x/globalfee module state from the consensus version 1 to
+// version 2. Specifically, it takes the parameters that are currently stored
+// and managed by the x/params modules and stores them directly into the x/globalfee
+// module state.
+func Migrate(
+	_ sdk.Context,
+	store sdk.KVStore,
+	_ exported.Subspace,
+	cdc codec.BinaryCodec,
+) error {
+	// var currParams types.Params
+	// legacySubspace.GetParamSet(ctx, &currParams)
+
+	denom, err := sdk.GetBaseDenom()
+	if err != nil {
+		denom = "ujuno"
```

Comment:
Probably is it better to use const types instead of hardcoding "juno":

https://github.com/CosmosContracts/juno/blob/reece/v47-globalfee-rewrite/app/params/const.go

Reply:
Yes good call

This logic is here for ujunox on testnet, still need to test this with a e2e on another denom. Cleaning this up and then checking upgrade still works

Reply:
Resolved in https://github.com/CosmosContracts/juno/pull/708/commits/151bbc384236df90cadbd785a903d2b305caf7c4

Reply:
Actually resolved with 863a1ab0626b1d46327c751e26d1a137db3c5d53 now.